pubspec.yaml

```diff
@@ -7,7 +7,7 @@ version: 0.3.0
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
-  flutter: ^3.3.0
+  flutter: ^3.0.0
```

Comment:
NixOs didn't package latest flutter yet, so I couldn't build the project. Feel free to remove this, if you want.